plugins/zammad-mcp/skills/zammad-mcp-issue-digest/scripts/triage_common.py

```
@@ -0,0 +1,683 @@
+    for label in item.get("labels") or []:
+        name = label.get("name") if isinstance(label, dict) else label
+        if name:
+            out.append(str(name))
+    return sorted(set(out), key=str.casefold)
+
+
+def comments_text(item: dict[str, Any]) -> str:
+    chunks = []
+    for comment in item.get("comments") or []:
+        if isinstance(comment, dict):
+            chunks.append(str(comment.get("body") or ""))
```

Ruff (0.15.12)

[warning] Use a list comprehension to create a transformed list

(PERF401)